www/src/components/container/side-nav.jsx

```diff
@@ -9,7 +9,26 @@ import { ScrollMarkerLink } from 'react-scroll-marker';
 import styles from './side-nav.module.scss';
 
 const SideNav = ({ children }) => (
-    <ul className={`flex-1 overflow-y-auto pb3 ${styles.sideNav}`}>{children}</ul>
+    <div className="flex-1 flex overflow-y-auto flex-column">
+        <ul className={`pb3 ${styles.sideNav}`}>{children}</ul>
+        <ul className="mt-auto ph3 pv3 flex">
+            <li className={styles.sideNavBottomFooterLinks}>
+                <Text
+                    href="https://github.com/thumbtack/thumbprint/blob/master/LICENSE"
```

Comment:
This link isn't working for me in the deploy preview- is that just a quirk of the preview, or a real issue?

Reply:
Whoops, I did it wrong. Fixed.